__tests__/app.test.js

```diff
@@ -178,6 +169,7 @@ describe("GET /api/articles", () => {
         expect(articles).toHaveLength(12);
         expect(articles[0]).toBeInstanceOf(Object);
         articles.forEach((article) => {
+          expect(article).toBeInstanceOf(Object);
```

Comment:
These sorts of assertion are not hurting anyone! But the fact that you assert the properties on the objects also covers that it is indeed an object